include/circt/Dialect/OM/OMAttributes.td

```diff
@@ -34,4 +34,22 @@ def ReferenceAttr : AttrDef<OMDialect, "Reference", [TypedAttrInterface]> {
   }];
 }
 
+def SymRefAttr : AttrDef<OMDialect, "SymRef", [TypedAttrInterface]> {
+  let summary = "An attribute that wraps a FlatSymbolRefAttr type";
+
+  let mnemonic = "symRef";
```

Comment:
Well, since you invited bikeshedding... I prefer `sym_ref` with snake_case for the mnemonic in IR, and keep camelCase only for C++ code.

Reply:
I agree, that's the convention, updated to `sym_ref`.